exampleCourse/questions/elementOverlayFigure/info.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Can we rename this question to just plain `elementOverlay`, as the basic example of using the element?